test/cbus_elixir_web/controllers/page_controller_test.exs

```diff
@@ -6,11 +6,11 @@ defmodule CbusElixirWeb.PageControllerTest do
     conn = get conn, "/"
     assert html_response(conn, 200) =~ "What is Columbus Elixir"
   end
-
+@tag :skip
```

Comment:
This test is broken? Hmm, Jones and I wrote that one. We thought it was written in such a way to avoid it from breaking if the tests run in the future as the code uses 'TODAY' to know which meeting is next.

Reply:
The test will break if you first ran he proj after april. The seed file was set to create meetings with Timex.now so If you started the proj brand new today only meetings from sept 18 on would be seed and that would break the test which is testing for a meeting in april '18

If you have never dropped your test db with test would still pass. However if you started fresh today it would fail.